live-build/misc/migration-scripts/dx_apply

```diff
@@ -24,6 +24,13 @@
 
 set -o pipefail
 
+#
+# Reset the umask to the default value. When called from the app-stack the
+# umask is set to 0027. Since those scripts can be called manually, we want
+# to have a consistent result regardless of the caller.
+#
+umask 0022
```

Comment:
Feel free to drop this if you think it is unnecessary but I was wondering if we should add
a comment explaining why this matter (e.g. we are creating directories that will need to
be accessed by multiple users in linux and we need the appropriate permissions, like `/export/home`).

Reply:
I agree, and think it's important. Otherwise, I'd be inclined to ask, why not change the umask for the default shell to be `0027`, to match the application?

An alternative solution to this would be, we could add some verification of the permissions for certain directories to the delphix-platform ansible logic. This would be more explicit and allow us to comment on why we need the specific permissions that are being set via ansible; was this considered?

I'm fine with this, just trying to weigh the different available options.

Reply:
I've put `0022` because that's the usual behaviour that people are expecting (i.e. the ability to traverse directories for `others`). I think that if some files need to have their access restricted, I'd rather see it done explicitly.

I did look at `delphix-platform` initially, however since I wanted to make this change for consistency anyway I thought that a delphix-platform change wasn't required anymore.